ApnsPHP/Push.php

```diff
@@ -638,6 +638,12 @@ private function httpSend(Message $message, &$reply): bool
             CURLOPT_POSTFIELDS => $message->getPayload()
             ]) && ($reply = curl_exec($this->hSocket)) !== false)
         ) {
+            $errorCode = curl_errno($this->hSocket);
+
+            if ($errorCode > 0) {
+                $this->logger->error(sprintf('HTTP request error code: %d; message: %s.', $errorCode, curl_error($this->hSocket)));
```

Comment:
Let's use the logger context here:
```suggestion
                $this->logger->error('HTTP request error code: {code}; message: {message}.', ['code' => $errorCode, 'message' => curl_error($this->hSocket)]);
```

Reply:
Added context